user-guide/scheduler.rst

```diff
@@ -608,7 +608,7 @@ specify other types of MPI task placement. For example, setting it to
 
     module load xthi
     export MPICH_RANK_REORDER_METHOD=0
-    srun --nodes=8 --tasks-per-node=2 xthi
+    srun --cpu-bind=core xthi
```

Comment:
It should be _--cpu-bind=cores_

Reply:
Fixed in https://github.com/ARCHER2-HPC/archer2-docs/pull/48/commits/e5b33cb6bab8e9f6412ff58249049a5d0cab48bf